articles/aks/deploy-spinkube.md

```diff
@@ -0,0 +1,253 @@
+kubectl annotate node --all kwasm.sh/kwasm-node=true
+```
+
+After annotating the Kubernetes node(s), `runtime-class-manager` will use a Kubernetes *Job* to modify the desired node(s). After successful deployment of `containerd-shim-spin`, the node(s) will be labeled with a `kwasm.sh/kwasm-provisioned` label. 
+
+### Deploy the `spin-operator`
+
+The `spin-operator` consists of two Custom Resource Definitions (CRDs), the RuntimeClass for `spin`, and a `SpinAppExecutor`, which must be deployed to your AKS cluster. 
+
+Start by deploying the CRDs and the RuntimeClass for `spin`:
+
+```bash
+kubectl apply -f https://github.com/spinkube/spin-operator/releases/download/v0.2.0/spin-operator.crds.yaml
+kubectl apply -f https://github.com/spinkube/spin-operator/releases/download/v0.2.0/spin-operator.runtime-class.yaml
+```
+
+The `spin-operator` itself is deployed using a Helm chart:
+
+```bash
+helm install spin-operator --version 0.2.0 \
+  --namespace spin-operator --create-namespace \
+  --wait oci://ghcr.io/spinkube/charts/spin-operator
+```
+
+Finally, create a `SpinAppExecutor` in the default namespace:
+
+```bash
+kubectl apply -f https://github.com/spinkube/spin-operator/releases/download/v0.2.0/spin-operator.shim-executor.yaml
+```
+
+## Run a Spin App on AKS
+
+In this section you will verify the SpinKube installation, by creating a simple Spin App using the `spin` CLI and JavaScript. Start by creating a new Spin App using the `http-js` template:
+
+```bash
+# Create a new Spin App
+spin new -t http-js --accept-defaults hello-spinkube
+
+# Move into the application folder
+cd hello-spinkube
+
+# Install Wasm Tooling
+npm install
+```
+
+The `spin` CLI create a basic *Hello, World* application, package and push the Spin App without further modifications to a OCI compliant container registry:
+
+```bash
+# Compile the app to Wasm
+spin build
+
+# Package and Distribute the Spin App
+spin registry push ttl.sh/hello-spinkube:0.0.1
+```
+
+> Both `spin` CLI and SpinKube integrate seamlessly with private container registries such as Azure Container Registry (ACR). To authenticate the `spin` CLI use `spin registry login` and supply corresponding credentials. SpinKube can pull OCI artifacts from ACR using underlying Azure identities with `AcrPull` permissions for the desired ACR instance(s).
+
+By using the `spin kube scaffold` command, you can create necessary Kubernetes deployment manifests. You can deploy those to Kubernetes using your preferred tooling:
+
+```bash
+# Create Kubernetes Deployment Manifests
+spin kube scaffold --from ttl.sh/hello-spinkube:0.0.1 > spinapp.yaml
+
+# Deploy the Spin App to AKS
+kubectl apply -f spinapp.yaml
+```
+
+### Explore the Spin App in AKS
+
+Having an Spin App deployed to the AKS cluster, you can explore different objects being created. 
+
+You can retrieve the list of Spin Apps using `kubectl get spinapps`
+
+```bash
+# Get all Spin Apps in the default namespace
+kubectl get spinapps
```

Comment:
Perhaps, for commands like kubectl where you want to show output and be able to execute the command interactively via cloud shell, consider the following:

\```azurecli-interactive
kubectl get spinapps
\```

\```OUTPUT
NAME             TYPE        CLUSTER-IP    EXTERNAL-IP   PORT(S)   AGE
 hello-spinkube   ClusterIP   10.43.35.78   <none>        80/TCP    24s
\```

Reply:
Addressed as part of 4d6a07f